descriptors/ModuleDescriptor-template.json

```diff
@@ -189,6 +189,24 @@
         }
       ]
     },
+    {
+      "id": "inventory-event-handlers",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/inventory/handlers/created-srs-marc-bib-record",
+          "permissionsRequired": [
+            "inventory.events.post"
```

Comment:
My understanding is that this permission is required for any client invoking this endpoint. How will mod-pubsub, which I believe will be invoking it, be granted that permission?

Reply:
https://github.com/folio-org/mod-pubsub#permissions

Reply:
@OleksiiKuzminov Thank you for that information.

Permissions are usually specific to an endpoint, why is this generalised to any events handling endpoint?

Reply:
That information suggests that a special pub sub user needs to be set up with the appropriate permissions, for each tenant. Meaning that someone needs to know what about all of the event subscribers in the system.

Have I followed that documentation correctly?

Reply:
For the first version of the securing mechanism, you are right. But we have an umbrella and spike to investigate the best way of securing HTTP calls through pubsub. For now, we create a special user for pubsub and register it TenantApi. After it, we assign all pub-sub handlers permissions for this user